templates/node.js-express-webapp-to-linux-on-azure.properties.json

```diff
@@ -8,9 +8,19 @@
         },
         {
             "name": "webAppName",
-            "type": "string",
+            "type": "dataSourcePicklist",
             "required": "true",
-            "displayName":"Web App name"
+            "displayName": "Web App name"
+        }
+    ],
+    "dataSourceBindings": [
+        {
+            "dataSourceName": "AzureRMWebAppNamesByAppType",
+            "endpointParameterName": "azureRmConnection",

```

Comment:
Won't `endpoint` suffice?

Reply:
In task inputs, `endpointId` is present, which is the input corresponding to the endpoint (The value of which would be the endpoint id).
Named it `endpointParameterName` for clarity, since in future we may introduce `endpointId` as well.

Reply:
I suggest we go with `endpoint` and use a resolver to get the id. 
In your case `{{ azureRmConnection }}` maybe?